content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -103,38 +110,38 @@ Upgrading the Istio CNI agent to a compatible version in-place will not disrupt
 $ helm upgrade istio-cni istio/cni -n istio-system
 {{< /text >}}
 
-### Upgrade the Gateway component (optional)
+## Upgrade waypoints and gateways using tags
 
-Gateway components manage east-west and north-south dataplane traffic between ambient mesh boundaries, as well as some aspects of the L7 dataplane.
+If you have followed best practices, all of your gateway, workload, and namespaces use either the default revision (which is effectively a tag named default), or the `istio.io/rev` label with the value set to a tag name. We will now upgrade all of these data planes to the new version of Istio by moving their tags to point to the new version, one at a time. To list all tags in your cluster, run:
 
-{{< text syntax=bash snip_id=upgrade_gateway >}}
-$ helm upgrade istio-ingress istio/gateway -n istio-ingress
+{{< text syntax=bash snip_id=list_tags >}}
+$ kubectl get mutatingwebhookconfigurations -l 'istio.io/tag' -L istio\.io/tag,istio\.io/rev
 {{< /text >}}
 
-## Configuration
+For each tag, you can upgrade the tag by running the following command, replacing `$MYTAG` with your tag name, and `$REVISION` with your revision name:
 
-To view supported configuration options and documentation, run:
-
-{{< text syntax=bash snip_id=show_istiod_values >}}
-$ helm show values istio/istiod
+{{< text syntax=bash snip_id=upgrade_tag >}}
+$ helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags="{$MYTAG}" --set revision="$REVISION" -n istio-system | kubectl apply -f -
 {{< /text >}}
 
-## Verify the installation
-
-### Verify the workload status
-
-After installing all the components, you can check the Helm deployment status with:
+This will upgrade all dataplanes referencing that tag, except for those using [manual gateway deployment mode](docs/tasks/traffic-management/ingress/gateway-api/#manual-deployment), which are dealt with below, and sidecars, which are not used in Ambient mode. It is recommended that you closely monitor the health of applications using the upgraded dataplane before upgrading the next tag. If you detect a problem, you can rollback a tag, resetting it to point to the name of your old revision:
```

Comment:
```suggestion
This will upgrade all objects referencing that tag, except for those using [manual gateway deployment mode](docs/tasks/traffic-management/ingress/gateway-api/#manual-deployment), which are dealt with below, and sidecars, which are not used in ambient mode. 

It is recommended that you closely monitor the health of applications using the upgraded data plane before upgrading the next tag. If you detect a problem, you can rollback a tag, resetting it to point to the name of your old revision:
```